tests/mail/tests.py

```diff
@@ -1270,6 +1305,19 @@ def test_server_open(self):
         backend.close()
         self.assertTrue(opened)
 
+    def test_server_login(self):
+        """
+        Even if the Python SMTP server doesn't support authentication, this
```

Comment:
", the login process starts and the appropriate exception..." (chop "this checks that")